### ISSUE TITLE
docker server with postgres, root.user missed

### ISSUE THREAD

**Issue by gastonstrizzolo** [Ericsson/codechecker#3525]
**Describe the bug**
I cloned the repository, then `make package` as the manual indicates, after that `docker build -t codechecker-web:latest web/docker` executes well but when I tried `docker-compose -f web/docker/services/docker-compose.psql.yml` it has the following error:
```
Traceback (most recent call last):
codechecker_1  |   File "/codechecker/lib/python3/codechecker_common/cli.py", line 206, in main
codechecker_1  |     sys.exit(args.func(args))
codechecker_1  |   File "/codechecker/lib/python3/codechecker_server/cmd/server.py", line 435, in __handle
codechecker_1  |     main(args)
codechecker_1  |   File "/codechecker/lib/python3/codechecker_server/cmd/server.py", line 1008, in main
codechecker_1  |     server_init_start(args)
codechecker_1  |   File "/codechecker/lib/python3/codechecker_server/cmd/server.py", line 986, in server_init_start
codechecker_1  |     environ)
codechecker_1  |   File "/codechecker/lib/python3/codechecker_server/server.py", line 972, in start_server
codechecker_1  |     root_sha = __make_root_file(root_file)
codechecker_1  |   File "/codechecker/lib/python3/codechecker_server/server.py", line 948, in __make_root_file
codechecker_1  |     with open(root_file, 'w', encoding="utf-8", errors="ignore") as f:
codechecker_1  | PermissionError: [Errno 13] Permission denied: '/workspace/root.user'
services_codechecker_1 exited with code 1
```
What I have tried next was create the `root.user` file empty with execution-read-write permissions, in a ./workspace directory, but the error persists, would you help me?

**CodeChecker version**
6.19.0 

**To Reproduce**
`docker-compose -f web/docker/services/docker-compose.psql.yml`

**Expected behaviour**
The server running a postgres database

**Desktop (please complete the following information)**
Ubuntu 20.04

**Comment by whisperity** [comment]
I've just been tinkering with running a CodeChecker server in a Docker container. Are you running the `/workspace` volume mounted to something on your host machine? In that case, you need to `chown` or `chmod` the files so that the user running *inside* the Docker can access them. I think `UID` and `GID` are both `950` by default. Alternatively, you can either specify explicitly in which user's (and group's) name the container should be running as (preferably yours, as you have permissions to the mounted volume), or build the image and give these numeric parameters to `docker build` itself.

**Comment by gastonstrizzolo** [comment]
Thanks! The solution was referred to the first, I was looking in my local /workspace folder and not in the mounted one :)

**Comment by whisperity** [comment]
> Thanks! The solution was referred to the first, I was looking in my local /workspace folder and not in the mounted one :)

@gastonstrizzolo: So it's "fixed"? There is a similar issue, #3458 which might indicate this is a documentation error.
